app/assets/javascripts/active_admin/base.js

```diff
@@ -345,184 +427,81 @@
-    $("body").trigger("modal_dialog:before_open", [ form ]);
-    form.dialog({
-      modal: true,
-      open: function open(event, ui) {
-        $("body").trigger("modal_dialog:after_open", [ form ]);
-      },
-      dialogClass: "active_admin_dialog",
-      buttons: {
-        OK: function OK() {
-          callback($(this).serializeObject());
-          $(this).dialog("close");
-        },
-        Cancel: function Cancel() {
-          $(this).dialog("close").remove();
+    };
+    PerPage._jQueryInterface = function _jQueryInterface(config) {
+      return this.each(function() {
+        var $this = $(this);
+        var data = $this.data("perPage");
+        if (!data) {
+          data = new PerPage(this);
+          $this.data("perPage", data);
         }
-      }
-    });
-  };
-  (function($, ActiveAdmin) {
-    var PerPage = function() {
-      function PerPage(element) {
-        this.element = element;
-      }
-      var _proto = PerPage.prototype;
-      _proto.update = function update() {
-        var params = ActiveAdmin.queryStringToParams().filter(function(_ref) {
-          var name = _ref.name;
-          return name != "per_page" || name != "page";
-        });
-        params.push({
-          name: "per_page",
-          value: this.element.value
-        });
-        if (ActiveAdmin.turbolinks) {
-          ActiveAdmin.turbolinksVisit(params);
-        } else {
-          window.location.search = ActiveAdmin.toQueryString(params);
+        if (config === "update") {
+          data[config]();
         }
-      };
-      PerPage._jQueryInterface = function _jQueryInterface(config) {
-        return this.each(function() {
-          var $this = $(this);
-          var data = $this.data("perPage");
-          if (!data) {
-            data = new PerPage(this);
-            $this.data("perPage", data);
-          }
-          if (config === "update") {
-            data[config]();
-          }
-        });
-      };
-      return PerPage;
-    }();
+      });
+    };
+    return PerPage;
+  }();
+  (function($) {
     $(document).on("change", ".pagination_per_page > select", function(event) {
       PerPage._jQueryInterface.call($(this), "update");
     });
     $.fn["perPage"] = PerPage._jQueryInterface;
     $.fn["perPage"].Constructor = PerPage;
-  })(jQuery, ActiveAdmin);
-  ActiveAdmin.TableCheckboxToggler = function(_ActiveAdmin$Checkbox) {
-    _inheritsLoose(TableCheckboxToggler, _ActiveAdmin$Checkbox);
+  })(jQuery);
+  function _inheritsLoose(subClass, superClass) {
+    subClass.prototype = Object.create(superClass.prototype);
+    subClass.prototype.constructor = subClass;
+    subClass.__proto__ = superClass;
+  }
+  var TableCheckboxToggler = function(_CheckboxToggler) {
+    _inheritsLoose(TableCheckboxToggler, _CheckboxToggler);
     function TableCheckboxToggler() {
-      return _ActiveAdmin$Checkbox.apply(this, arguments) || this;
+      return _CheckboxToggler.apply(this, arguments) || this;
```

Comment:
I think the rename of this parameter unrelated to the commit including it, so I would split the rename to a previous commit.

Reply:
This comes from changing `ActiveAdmin. CheckboxToggler = ...` to `export default CheckboxToggler;` which is in this same commit. That's on the compiled side, so no, for this one, it's one the right commit :)

Reply:
Right! :+1: